### PR TITLE
[Select] Cover palettes on icons / placeholders 

### PR DESCRIPTION
**Pull request by jeremie-lucca**
chrome_My59SwCCGh

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1536

File: packages/ng/style/definitions/select/_select-clearer.scss [hunk 1]
```diff
@@ -63,4 +55,24 @@
...
+				@include selectColoringClearer($name);
+			}
+		}
+	}
+}
+
+@mixin selectColoringClearer($palette) {
+		color: _color($palette, 700);
+
+		&:hover {
```
Comment (vvalentin-lucca) [inline packages/ng/style/definitions/select/_select-clearer.scss:74]: No focus management too?

File: packages/ng/style/definitions/select/_select-clearer.scss [hunk 2]
```diff
@@ -63,4 +55,24 @@
...
+			background:  _color($palette, 50);
```
[the remark this answers is not in the document]
Comment (jeremie-lucca) [reply packages/ng/style/definitions/select/_select-clearer.scss:75]: ✅

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1536

File: packages/ng/style/definitions/select/_select-clearer.scss (hunk 1 above)
Comment (vvalentin-lucca) [reply packages/ng/style/definitions/select/_select-clearer.scss:74]: Yes, it should be a focusable element, to be easily reached with keyboard and mouse.

Even if it's not the case yet, I think we can plan the CSS style accordingly.

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1536

**Review by vvalentin-lucca — APPROVED** [review]

**Pull request merged**